azurerm/validators.go

```diff
@@ -76,6 +77,122 @@ func validateIso8601Duration() schema.SchemaValidateFunc {
 	}
 }
 
+func validateAzureVirtualMachineTimeZone(acceptEmpty bool) schema.SchemaValidateFunc {
```

Comment:
since this is only used in a single place, we can assume this is the case - I'll push a commit to remove this.